src/components/tipRecords/TipComment.vue

```diff
@@ -59,6 +59,7 @@ export default {
   },
   methods: {
     goToCommentPage(tipId, id) {
+      if (this.$store.state.route.params.id === id) return;
```

Comment:
Won't it block navigation if we put TipComment component on a page that have `id` param the same as `id` of displayed comment? Shouldn't we also remove `cursor: pointer` at the same time?